CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.59.0] - 2023-12-12
```

Comment:
```suggestion
## [0.59.0] - 2023-12-21
```

Reply:
We can maybe also add a 🌲 to the entry :)

Reply:
Good catch!